ska_sync/ska_sync_config

```diff
@@ -50,6 +50,31 @@ file_sync:
   acq_stats:
     - acq_stats.h5
 
+  # Minimal set of eng-archive files for doing ACA and ACIS load review
```

Comment:
I also wasn't sure if opt in or opt out was the best strategy for ska_sync.  Either fine by me.